### DISCUSSION TITLE
WiFi Usage | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1688] [Q&A]
# WiFi Usage | Seeed Studio Wiki

WiFi usage with Seeed Studio XIAO ESP32C6.

https://wiki.seeedstudio.com/xiao_wifi_usage_esp32c6/

**Comment by Gryndii** [comment]
For any wifi examples on c6 I got this error in serial
assert failed: remove_free_block heap_tlsf.c:233 (prev && "prev_free field can not be null")

also I can't upload sketch without boot button very often

**Reply by Gryndii** [reply]
skip this, something wrong with my board as for another everything is ok

**Comment by LutzFassl** [comment]
My wifi is connecting fine when I use the ESP32C3 as is. But when I plug it in the Xiao Expansion Board wifi is not working. Why?

EDIT: I didn't change anything but updated from espressif esp32 library for Arduino IDE from v3.0.5 to v.3.0.6 - now it works.
